src/daos/docker/ceph/Dockerfile

```diff
@@ -11,6 +11,28 @@ LABEL maintainer="seagate-daos@seagate.com"
 
 # get & build the ceph source
 COPY ccache.conf /etc/ccache.conf
+
+# redhat.repo is only available from within Seagate's network and is needed since xmlstarlet was removed from @system repo
+COPY redhat.repo /etc/yum.repos.d/redhat.repo
+RUN sudo dnf clean all
+RUN sudo dnf makecache
+# redhat.repo end changes
+
+RUN sudo dnf install dnf-plugins-core -y
+RUN sudo dnf config-manager --set-enabled powertools
+RUN sudo dnf -y module enable javapackages-tools
+RUN sudo dnf install epel-release -y
+RUN sudo dnf install gcc-toolset-11 -y
+# xz is needed to decompress the tar file
+RUN sudo dnf install xz -y
+RUN sudo dnf install wget -y
+RUN bash -c "echo "source scl_source enable gcc-toolset-11" >> ~/.bashrc"
```

Comment:
Does this get run too?

Reply:
I had tested it on the SSC 1901 machine and it didn't fail.  When running it on the SSC 3251 machine it did fail.  I was just looking at it.  Good catch.  I also used the quotes incorrectly, I'm not sure what actually got appended